docusaurus/docs/dev-docs/migration/v4-to-v5/additional-resources/helper-plugin.md

```diff
@@ -7,7 +7,7 @@ tags:
   - upgrade to Strapi 5
 ---
 
-import DoNotMigrateYet from '/docs/snippets/_do-not-migrate-to-v5-yet.md'
+import DoNotMigrateYet from '/docs/snippets/\_do-not-migrate-to-v5-yet.md'
```

Comment:
```suggestion
import DoNotMigrateYet from '/docs/snippets/_do-not-migrate-to-v5-yet.md'
```

Probably added by an automated tool. I believe this is wrong and won't render the snippet if you add a `\`.

Reply:
I think it's the correct escape character for markdown, I've used it in a few places in the docs before. If we're not rendering it correctly somewhere, we should try to fix that!

Reply:
yeah prettier is doing its thing :)